challenge_demo/src/demo.py

```diff
@@ -179,7 +179,7 @@ def main():
         robot.lights.set_color(0, 0, 1)  # be sure lights are blue
 
         robot.head.look_at_standing_person()
-        for arm in robot.arms:
+        for arm in robot.arms.itervalues():
```

Comment:
@alberth is this the correct way of using this now?

Reply:
Eventually, I think 'reset' should be a robot function rather than a challenge function, but we're not even near there currently, so this is fine for now.